python_script/meter-health/meter_health.py

```diff
@@ -0,0 +1,131 @@
+# -*- coding: utf-8 -*-
+import requests
+import csv
+import logging
+logging.basicConfig(filename='example.log', level=logging.DEBUG)
+from oauthlib.oauth2 import LegacyApplicationClient
+from requests_oauthlib import OAuth2Session
+from config.credentials import *
+
+
+class MeterHealth:
+    # URL information for the API. ONLY change this method if you know what you are doing
+    BASE_URL = "http://54.90.173.204:8080"
+    ORGANIZATIONS = "/api/v2.json"
+    METERS = "/api/v2/organizations/%d/meters.json"
+    TOKEN_URL = "/oauth/token"
+
+    def __init__(self):
+        self.token = self.retrieve_token()
+        self.payload = {'access_token': self.token}
+
+    def create_report(self):
+        """ It is the main method that calls
+        all that need to be done
+        """
+        if self.token:
+            organizations = self.retrieve_organizations()
+            if organizations:
+                self.build_csv(organizations)
+            else:
+                print("There aren't organizations \n")
+
+    def build_csv(self, organizations):
+        # Creates/opens the file which will store the info
+        total_org = len(organizations)
+        print("We found %d organizations \n" % total_org)
+        with open(file_name, 'w+') as csv_file:
+            writer = csv.writer(csv_file)
+            writer.writerow(['Organization Name', 'Meter ID', 'Meter Name',
+                             'Meter Type', 'Meter Status', 'Last Reading'])
+            self.add_information_to_csv({'organizations': organizations,
+                                         'writer': writer,
+                                         'total_org': total_org})
+            print("Process successfully finished, analyzed a total of %d organizations" % total_org)
```

Comment:
For interpolation in strings is better to use .format() function, for example:
"We found {0} organizations".format(total_org)

% is an old way to format a string.